vendor/vendor.json

```diff
@@ -39,884 +39,892 @@
 			"revisionTime": "2017-07-27T15:54:43Z"
 		},
 		{
-			"checksumSHA1": "y1bmVkrw6lKwgq8z+S3GzO1F5JE=",
+			"checksumSHA1": "N+8JQQqczoq0UfwXrcVqqRDcb5E=",
```

Comment:
Did you accidentally govendor the root? I think it'll complain about that. See any of the recent `Deps: Bump aws-sdk-go` PRs for how it's done (we're hoping to automate this shortly)

Reply:
thanks, I will split that out